packages/augur-ui/src/modules/trading/components/wrapper/wrapper.styles.less

```diff
@@ -24,6 +24,10 @@
     &.Full {
       flex-grow: unset;
     }
+
+    &.Liquidity {
+      flex-grow: inherit;
```

Comment:
you can just say 1, don't need to put inherirt

Reply:
unset doesn't work

Reply:
flex-grow: 1 should work

Reply:
removed it.